### ISSUE TITLE
Codes about FeatureAlign_V2

### ISSUE THREAD

**Issue by LightningChan** [EMI-Group/FaPN#6]
In module[ FeatureAlign_v2](https://github.com/EMI-Group/FaPN/blob/main/detectron2/modeling/backbone/fan.py)
`offset = self.offset(torch.cat([feat_arm, feat_up * 2], dim=1))  # concat for offset by compute the dif`
Why multiple feat_up by 2?

**Comment by ShihuaHuang95** [comment]
We hope to enlarge the magnitude of upsampled features in case of suppression.

**Comment by 2653888596** [comment]
Thank you！ I also have the same question！

**Comment by yukilulu** [comment]
thank reply!
